client/src/components/FrontWarmFuzzy.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const FrontWarmFuzzy = (props) => {
+  const { title, handleClick } = props
+  return (
+    <div onClick={handleClick} className="p-2 m-3 text-center align-middle place-self-center">
+      <p> <span className="font-bold">Title:</span> {` ${title}`}</p>
```

Comment:
avoid the unnecessary string template if possible - backticks shouldn't be required